lang/python.talon

```diff
@@ -113,3 +113,11 @@ action(user.code_state_return):
 dunder in it: insert("__init__")
 state (def | deaf | deft): "def "
 pie test: "pytest"
+
+# show and print python functions
+list python: user.list_python_functions()
+hide python: user.hide_python_functions()
```

Comment:
One thing that might be worth adding is the ability to select the function by index when imgui is open.

Similar to what’s done with the hello system when open perhaps.
https://github.com/knausj85/knausj_talon/blob/eeea6454aa217aa5f24d0eb36885c639a439d87c/misc/help_open.talon#L5

I’ve found selecting via numbers VERY convenient fast and reliable. Could have a command like
`fun <number>: `

Could also be done later, but I’d be happy to help. What do you think?